src/loaders/compute_tools/eggnog/eggnog.py

```diff
@@ -22,7 +21,7 @@ def _run_eggnog_single(
         threads_per_tool_run: int,
         debug: bool) -> None:
 
-    metadata_file = output_dir / EGGNOG_METADATA
+    metadata_file = output_dir / TOOL_METADATA
```

Comment:
Why does eggnog already have a metadata file while checkm2 doesn't?

Reply:
I believe we have metadata files for individual executable tools that can help us identify successful `data_ids` runs. But for batched tools (`checkm2`, `gtdb_tk`), the metadata file won't indicate which `data_ids` failed so we never add one.

Reply:
Isn't here a file which indicates which data ids have been run? I seem to remember something like that?

Reply:
for batched tools? I don't think so. There is a failure file created after the tool has been run to make sure all result files are in place. But if the tool execution failed, we have no way to identify which id has been run or not.

Reply:
ok, it's been a long time since I looked at this code and I'm not super familiar with it anyway.

There's no code to look at partial results and figure out if any are successful? Is that even possible, or for all the tools if the execution didn't finish normally the whole batch is dead?

Reply:
It just never became a priority for us to develop a strategy for handling reruns and failures. There is some logic to verify that result files are in place. For failures, the parser will simply skip them. However, no rerun logic has been implemented yet.

Reply:
:+1: